backend/substrapp/tasks/tasks.py

```diff
@@ -227,25 +227,21 @@ def fetch_model(channel_name, parent_tuple_type, authorized_types, input_model,
 def fetch_models(channel_name, tuple_type, authorized_types, input_models, directory):
 
     models = []
```

Comment:
@Kelvin-M just to be clear, my suggestion is to use a function call (not a thread) when there's only one model to download:


```suggestion
    if len(input_models) == 1:
        # Don't create sub-processes if there's only one model to download
        fetch_model(channel_name, tuple_type, authorized_types, input_model, directory)
        return

    models = []
```

Reply:
Thank you :)
As answered [here](https://github.com/SubstraFoundation/substra-backend/pull/351#issuecomment-733540836), I would prefer to use the same solution for all cases than handle them with a special behavior for each (here 2, but we can do an analysis to know when it's better to use sequential calls or parallel calls) as it's not "slower" to spawn a Process

Reply:
I just ran a quick benchark and process creation takes 6ms on my machine, nothing to argue about here :)